modules/ROOT/pages/cloudhub-deploy-options.adoc

```diff
@@ -3,21 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Message flows from Partner Manager are deployed by default to the CloudHub region specified in the business group’s default settings within Access Manager at the time of first deployment from Partner Manager.
-Subsequent deployments get deployed to the same region as the first deployment.
+Partner Manager message flows are initially deployed by default to the CloudHub region specified in the default
+settings of Access Manager's business group. Subsequent deployments are deployed to the same region as the first deployment.
 
 To access the default deployment settings for your organization, navigate to *Access Management > Partner Manager*.
 
 You can configure the following Partner Manager Settings for CloudHub deployment.
```

Comment:
Also, search this PR for "configur" and evaluate whether it should be replace with "customiz", because I think that customization makes more clear that you are altering a default config that already exists.